### PR TITLE
SY-922 Fix Create Range from Line Plot Context Menu

### PR DESCRIPTION
**Pull request by pjdotson**
# Fix Pull Request Template

## Key Information

- **Linear Issue**: [SY-922](https://linear.app/synnaxlabs/issue/SY-922/fix-creating-range-from-line-plot-context-menu)

## Description

Removed the buffer from the range console and allowed ranges to be passed directly through the create range window layout form.

## Basic Readiness Checklist

- [x] I have performed a self-review of my code.
- [x] I have added sufficient regression tests to cover the changes.

## Manual QA Additions

- [x] I have updated the [Release Candidate](/.github/PULL_REQUEST_TEMPLATE/rc.md) template
  with necessary manual QA steps to test my change.

### CODE REVIEW

File: console/src/range/migrations/v0.ts [hunk 1]
```diff
@@ -41,15 +45,13 @@ export const sliceStateZ = z.object({
   version: z.literal("0.0.0"),
   activeRange: z.string().nullable(),
   ranges: z.record(rangeZ),
-  buffer: staticRangeZ.partial().nullable(),
```
Comment (pjdotson) [inline console/src/range/migrations/v0.ts:44]: @emilbon99 Should I create a v1.ts file with the `buffer` field removed from the `SliceState`?

Comment (emilbon99) [reply console/src/range/migrations/v0.ts:44]: No need. I think it's fine if we leave that as stranded state.

File: console/src/range/selectors.ts [hunk 2]
```diff
@@ -19,21 +23,12 @@ import { SLICE_NAME, type SliceState, type StoreState } from "@/range/slice";
 const selectState = (state: StoreState): SliceState => state[SLICE_NAME];
 
 /**
- * Selects ranges with the given keys. If no keys are provided, all ranges are selected.
```
Comment (emilbon99) [inline console/src/range/selectors.ts:22]: Why did we get rid of this?

Comment (emilbon99) [reply console/src/range/selectors.ts:22]: Oh it was moved to a different spot. Ok

**Review by emilbon99 — APPROVED** [review]
LGTM

**Review by Lham42 — APPROVED** [review]

**Pull request merged**